lms/djangoapps/courseware/exceptions.py

```diff
@@ -25,3 +25,19 @@ class CourseAccessRedirect(Redirect):
     def __init__(self, url, access_error=None):
         super(CourseAccessRedirect, self).__init__(url)
         self.access_error = access_error
+
+
+class CourseRunNotFound(ValueError):
```

Comment:
Subclassing `ValueError` here in order to avoid breaking existing users of `get_course` that check for a `ValueError`.